### ISSUE TITLE
Client initialization timeouts

### ISSUE THREAD

**Issue by Christopher-Bradshaw** [launchdarkly/go-client#87]
Hi all,

Since Monday 10/10 at ~10:15GMT any attempt to initialize connect the LD client with something like:

```
launchdarkly "github.com/launchdarkly/go-client"
launchdarkly.MakeClient(config.LDApiKey, 6*time.Second)
```

Has been failing with `Timeout encountered waiting for LaunchDarkly client initialization`. Other clients appear to be working fine. I have pushed the timeout as high as 20 seconds with no improvement. Trying with an invalid api key quickly gets a 401 response.

This is running the latest version of the client, and the start of the failures does not appear to correspond with deploys or config changes on our end.

Any suggestions?

**Comment by drichelson** [comment]
@Christopher-Bradshaw do you mind emailing support@launchdarkly.com and I can help right away?

**Comment by Christopher-Bradshaw** [comment]
Thanks for the quick response. Emailed support with the issue.

**Comment by drichelson** [comment]
Fixed in #88
